### ISSUE TITLE
mineral and ion content related trait revision (TO:0000465)

### ISSUE THREAD

**Issue by austinmeier** [Planteome/plant-trait-ontology#373]
revised the definition of this term

old: The total mineral and/or ion content measured in a plant or a plant part.

New: A biochemical trait (TO:0000277) that is the amount of a particular mineral or ion constituent occurring in a plant anatomical entity (PO:0025131).

**Comment by cooperl09** [comment]
Propose rename class  to “chemical entity-related trait”

chemical entity (CHEBI:24431):  A chemical entity is a physical entity of interest in chemistry including molecular entities, parts thereof, and chemical substances.

Propose revised def’n: 
Currently: 
A biochemical trait (TO:0000277) that is the amount of a particular mineral or ion constituent occurring in a plant anatomical entity (PO:0025131).

Proposed: A biochemical trait (TO:0000277) which is a quality of a chemical entity (CHEBI:24431) that is part of a cellular component (GO:0005575) or a plant anatomical entity (PO:0025131).

Comment: chemical entity (CHEBI:24431) includes atoms (CHEBI_33250), molecular entities (CHEBI_23367), such as ions (CHEBI_24870), and chemical substances (CHEBI_59999).

**Comment by austinmeier** [comment]
Agreed, this works better because of the ChEBI definitions of "mineral" doesn't quite fit here.
